docs/tutorials/integrations/kserve/kserve-custommetrics-one.md

```diff
@@ -0,0 +1,117 @@
+---
+template: main.html
+---
+
+# Single Model Validation
+
+This tutorial shows how easy it is validate SLOs for a single model in [KServe](https://kserve.github.io/website/0.10/) when fetching metrics from a metrics database like Prometheus. We show this using the `sklearn-iris` model used for a [first `InferenceService`](https://kserve.github.io/website/master/get_started/first_isvc/) in the KServe documentation.
+
+???+ "Before you begin"
+    1. Try [your first experiment](../../../getting-started/your-first-experiment.md). Understand the main [concepts](../../../getting-started/concepts.md) behind Iter8 experiments.
+    2. Ensure that you have the [kubectl](https://kubernetes.io/docs/reference/kubectl/) CLI.
+    3. Have access to a cluster running [KServe](https://kserve.github.io/website). You can create a [KServe Quickstart](https://kserve.github.io/website/0.10/get_started/#before-you-begin) environment as follows:
+    ```shell
+    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.10/hack/quick_install.sh" | bash
+    ```
+    These instructions can be used to [install Prometheus](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#install-prometheus).
+
+
+## Experiment Setup
+
+Deploy the model and generate load against it. We follow the instructions for the [KServe first InferenceService](https://kserve.github.io/website/master/get_started/first_isvc) to deploy the model.
+
+### Create InferenceService for Initial Model
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model"
+EOF
+```
+
+### Generate Load
+
+Port forward requests to the cluster:
+
+```shell
+INGRESS_GATEWAY=$(kubectl get svc --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}')
+kubectl port-forward --namespace istio-system svc/$INGRESS_GATEWAY 8080:80
+```
+
+Finally, request predictions. Here we make a request about once a second.
```

Comment:
Perhaps, instead of "Finally, request predictions. Here we make a request about once a second." (given that the tutorial continues, finally seems a bit out of place), we can say ... 

> Send prediction requests to the inference service. The following script generates about one request a second. In a production cluster, this step is not required since your inference service will receive requests from real users.

Reply:
done